tests/main/security-seccomp/task.yaml

```diff
@@ -93,10 +93,14 @@ execute: |
     echo "and check that negative nice fails"
     test-snapd-setpriority -10 | MATCH 'Operation not permitted \(EPERM\)'
 
+    # TODO: filtering on setpriority is a bit confusing as it is not part
+    # of the "negative args" filter added in ec7c9f27c97 so the fact that
+    # negative args are denied is a bit magic
     echo "Explicitly deny arg filtered setpriority rule"
-    sed 's/^\(setpriority.*\)/~\1/g' "$SRC".orig > "$SRC"
+    cp -a "$SRC".orig "$SRC"
+    echo '~setpriority PRIO_PROCESS 0 >10' >> "$SRC"
     snapd.tool exec snap-seccomp compile "$SRC" "$BIN"
-    echo "and check that positive nice fails with explicit denial"
-    test-snapd-setpriority 10  | MATCH 'Insufficient privileges \(EACCES\)'
+    echo "and check that explicit requested fails with explicit denial"
```

Comment:
I am not sure about the grammar or meaning here.

Reply:
I tried to rephrase this